packages/language-server/src/__test__/completion.test.ts

```diff
@@ -1190,6 +1252,303 @@ suite('Completions', function () {
         },
       })
     })
+    test('@@index(fields: [|])', () => {
+      assertCompletion({
+        schema: /* Prisma */ `
+          model ThirdUser {
+              firstName String
+              lastName String
+              isAdmin Boolean @default(false)
+              @@index(field: [|])
+          }`,
+        expected: {
+          isIncomplete: false,
+          items: [
+            { label: 'firstName', kind: CompletionItemKind.Field },
+            { label: 'lastName', kind: CompletionItemKind.Field },
+            { label: 'isAdmin', kind: CompletionItemKind.Field },
+          ],
+        },
+      })
+    })
+    test('MongoDB: @@index([|])', () => {
+      assertCompletion({
+        provider: 'mongodb',
+        schema: /* Prisma */ `
+          type Address {
+            street String
+            number Int
+          }
+          model User {
+            id      Int     @id
+            email   String
+            address Address
+            @@index([|])
+          }`,
+        expected: {
+          isIncomplete: false,
+          items: [
+            { label: 'id', kind: CompletionItemKind.Field },
+            { label: 'email', kind: CompletionItemKind.Field },
+            { label: 'address', kind: CompletionItemKind.Field },
+          ],
+        },
+      })
+    })
+    test('MongoDB: @@index(ad[|])', () => {
+      assertCompletion({
+        provider: 'mongodb',
+        schema: /* Prisma */ `
+          type Address {
+            street String
+            number Int
+          }
+          model User {
+            id      Int     @id
+            email   String
+            address Address
+            @@index([a|])
+          }`,
+        expected: {
+          isIncomplete: false,
+          items: [
+            { label: 'id', kind: CompletionItemKind.Field },
+            { label: 'email', kind: CompletionItemKind.Field },
+            { label: 'address', kind: CompletionItemKind.Field },
+          ],
+        },
+      })
+    })
+    test('MongoDB: @@index(address[|])', () => {
+      assertCompletion({
+        provider: 'mongodb',
+        schema: /* Prisma */ `
+          type Address {
+            street String
+            number Int
+          }
+          model User {
+            id      Int     @id
+            email   String
+            address Address
+            @@index([address|])
+          }`,
+        expected: {
+          isIncomplete: false,
+          items: [
+            { label: 'id', kind: CompletionItemKind.Field },
+            { label: 'email', kind: CompletionItemKind.Field },
```

Comment:
OT: Does this really make sense here? I guess that is just how it works right now, but shouldn't it suggest a comma or nothing as there is nothing to auto complete? (Maybe we should add a TODO to the test or open an issue)

Reply:
Added a comment, I cannot reproduce when using VS Code
```
            // These are returned though the completion will actually be
            // No suggestions
            // TODO create a test that shows that
```